src-docs/src/views/header/header_stacked.js

```diff
@@ -52,9 +52,7 @@ export default () => {
         position={isFixed ? 'fixed' : 'static'}
         sections={[
           {
-            items: [
-              <OuiHeaderLogo iconType="logoElastic">Elastic</OuiHeaderLogo>,
-            ],
+            items: [<OuiHeaderLogo iconType="logoOUI">Elastic</OuiHeaderLogo>],
```

Comment:
Should references to Elastic in these files be changed to OpenSearch, or is that out of the scope of this PR?

Reply:
This should be OpenSearch UI, and this makes me think we actually need an OUI logo. The team has explored some options, we should put them forward

Reply:
@BSFishy @kgcreative elastic reference removal is in my PR, and the OUI logo is in an upcoming PR. The to-do list on the original issue is accurate.

Reply:
Additionally, there may be merge conflicts as I believe @bandinib-amzn originally included some of the elastic removal - I am not sure if those commits have been rolled back or not though. Will connect with @bandinib-amzn when I am not out of pocket.

Reply:
@KrooshalUX which elastic removal are we talking about? This PR includes only proposed changes from #111. Did I mistakenly remove other elastic reference? I'm not sure which commit needs to be rolled back.